### DISCUSSION TITLE
Where is the file saved?

### DISCUSSION THREAD

**Discussion by 1NEGROUP** [carson-katri/dream-textures#263] [General]
So I am Curious as to where the Generated images are saved? I just downloaded blender to use this.

**Comment by carson-katri** [comment]
They’re packed into the blend file. You can click File > Save a Copy to save them somewhere else.

**Reply by 1NEGROUP** [reply]
Thanks But I found two places where i could use the Image editor and i guess i wasn't using the program correctly. But I found out how to use it.
I was going to say you might put this [video](https://www.youtube.com/watch?v=5WboPlU9c6I) (its a video on how to use the program) in the usage area - because it wasnt very clear how to use seeing the pictures, especially if someone is going to download blender just for this program, which i did.

**Reply by carson-katri** [reply]
That video is somewhat outdated at this point.
